tests/components/weatherflow/test_config_flow.py

```diff
@@ -102,8 +102,8 @@ async def test_devices_with_mocks_timeout(
     """Test getting user input."""
 
     async def time_jump(hass: HomeAssistant):
-        async_fire_time_changed(hass, utcnow() + timedelta(seconds=10))
-        await asyncio.sleep(1)
+        async_fire_time_changed(hass, utcnow() + timedelta(seconds=100))
+        await asyncio.sleep(10)
```

Comment:
The CI has a limit 9s per test. We shouldn't be sleeping for more than a few microseconds in tests

Reply:
Is there a bettery way to catch an async-timeout on the coverage?

Reply:
patch the library to raise `asyncio.TimeoutError`

Reply:
Got it!